### PR TITLE
using [NotNull] attribute from the Common repo

### PR DESCRIPTION
**Pull request by Praburaj**
@muratg

### CODE REVIEW

File: src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs [hunk 1]
```diff
@@ -2,9 +2,10 @@
 #if DNX451
 using System.Runtime.Remoting;
 using System.Runtime.Remoting.Messaging;
-#endif
+#else
 using System.Threading;
```
Comment (muratg) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs]: System.Threading seems to be always included before this change.  Not it's not included for DNX451.  Why this change?

Comment (Praburaj) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs]: This was used only in the #else case. So VS intellisense is always showing this is an unused using statement. So moved it into the #else to clean up the intellisense warning.

Comment (Praburaj) [inline src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs]: and of course I verified project builds.

**Comment by muratg** [comment]
:shipit:
